llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2859,6 +2877,9 @@ class VPlan {
   /// Represents the loop-invariant VF * UF of the vector loop region.
   VPValue VFxUF;
 
+  /// Represents widened VF * UF for each UF of the vector loop region.
+  VPValue WidenVFxUF;
```

Comment:
To work with VPValue type inference , I create a recipe for VFxUF in #78309.
I thought similar challenge might arise with WidenVFxUF.
I'm seeking @fhahn's perspective on this matter.

Reply:
Agree, that will be great to have.

Reply:
Resolved in #80723.